support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -373,6 +373,8 @@ export const productCatalogDescription: Record<
 		benefitsMissing: [
 			appBenefit,
 			addFreeBenefit,
+			newsletterBenefit,
+			newsletterBenefitUS,
 			fewerAsksBenefit,
```

Comment:
What's the reason `newsletterBenefitUS` appears in both the `benefits` and `benefitsMissing` lists?

Reply:
The `US` has different wording in the newsletter benefit so `newsletterBenefitUS` will show for US and `newsletterBenefit` for everything else